Description: Set docs release to '' 
 The upstream source tries to set the release by using distutils. This
 will fail and is unneccessary.
Author: Christian Kellner <ck@xatom.net>

--- python-neuroshare-0.9.orig/doc/source/conf.py
+++ python-neuroshare-0.9/doc/source/conf.py
@@ -50,7 +50,7 @@ copyright = u'2013, Christian Kellner'
 # The short X.Y version.
 version = '1.0'
 # The full version, including alpha/beta/rc tags.
-release = pkg_resources.get_distribution("neuroshare").version
+release = ''
 
 # The language for content autogenerated by Sphinx. Refer to documentation
 # for a list of supported languages.
